gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -320,7 +293,7 @@ public static ZoneListOption pageSize(int pageSize) {
   /**
    * Class for specifying project options.
    */
-  class ProjectOption extends AbstractOption implements Serializable {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://www.codacy.com/app/mziccard/gcloud-java/file/2331164897/issues/source?bid=3123103&fileBranchId=3204660#l296)